extensions/amp-youtube/0.1/amp-youtube.js

```diff
@@ -94,6 +118,37 @@ class AmpYoutube extends AMP.BaseElement {
   }
 
   /** @private */
+  getPlayerReadyPromise_() {
+    return new Promise(resolve => {
+      this.getWin().addEventListener('message', event => {
+        const data = JSON.parse(event.data);
+        if (event.origin != 'https://www.youtube.com' ||
```

Comment:
Do this first and then parse. Otherwise you'll create lots and lots of errors from parsing non-JSON